.github/workflows/ci.yml

```diff
@@ -542,6 +542,65 @@ jobs:
             build/*.cmake
             build/testsuite/*/*.*
 
+  icx-vp2022:
+    name: "Linux VFXP-2022 icx/C++17 llvm13 py3.9 boost1.76 exr3.1 oiio2.3 avx2"
+    runs-on: ubuntu-latest
+    container:
+      image: aswftesting/ci-osl:2022-clang13
+    env:
+      CXX: icpx
+      CC: icx
+      CMAKE_CXX_STANDARD: 17
+      FMT_VERSION: 7.1.3
+      OPENIMAGEIO_VERSION: master
+      OPENIMAGEIO_CMAKE_FLAGS: -DBUILD_FMT_VERSION=7.1.3
```

Comment:
I believe the `FMT_VERSION` and `OPENIMAGEIO_CMAKE_FLAGS` lines are not necessary for icx, only icc.

Reply:
good catch, I'll try removing those.